Terraform files to deploy label-studio Google Cloud, cloudbuild.yaml (#1374)
heartexlabs/label-studio: docs/source/guide/install.md

pip install label-studio
```

## Install on Google Cloud

> 1. First login with `gcloud`

Reviewer: ```suggestion
1. Log in with `gcloud`
```

```bash
gcloud auth application-default login
```

> 2. You can use `direnv` to keep track of necessary env variables. Set up `.envrc` and don't forget to run `direnv allow`.

Reviewer: ```suggestion
> You can keep track of necessary env variables with `direnv`. Set up `.envrc` and don't forget to run `direnv allow`.
```

```bash

> export PROJECT_ID=maystra-poc

Reviewer: ```suggestion
export PROJECT_ID=<your-project>
```

```

> 3. Build the docker image and push it to Google Build with:

Reviewer: Does this command also build the Docker image? If no, please make these 2 separate steps:
```suggestion
3. Build the Docker image.
4. Push the Docker image you built to Google Build using the following command:
```
If this command does build the docker image, then minor suggestions:
```suggestion
3. Build the Docker image and push it to Google Build using the following command:
```

```bash
// Needs to be from the same directory as cloudbuild.yaml
gcloud builds submit
```

> 4. Init terraform and apply changes

Reviewer: ```suggestion
4. Initiate terraform and apply changes. From the command line, run the following:
```
